### DISCUSSION TITLE
Opi5plus - Boot from eMMC with / or /home on nvme

### DISCUSSION THREAD

**Discussion by tp0s** [Joshua-Riek/ubuntu-rockchip#489] [Q&A]
I know there's a few different ways to have boot dir on another disk/partition in a standard linux env but since this image (which is impressively stable btw, thanks so much for all your work, joshua) is preinstalled that option isn't viable (right? or did i miss something?). So i'm wondering if there's a way to have the eMMC do the booting, and then store / , or just /home on the nvme that's hooked into the m.2 slot? I know i can edit some boot files, or change /home after first time setup, or possibly clone the boot partition? but really wondering if there's an easier way from the get-go, or if altering boot / fstab / usermod -d is the way to go. 

or 

the eMMC was given to me by a friend who said that booting and running system procs would be faster and smoother, which is the only reason i'm interested in setting it up this way, i don't have much in-depth experience with flash storage outside of my router running openwrt. would i benefit more booting from nvme and utilizing the eMMC for things like dev/test/host web apps or running containers? i guess that should have been my first question, but i'm still interested in figuring out how to set up /system-boot on a different /dev than /  or  /home if it's feasible. thanks for your time!

**Comment by tp0s** [comment]
i made this way more difficult than it needed to be. i'm just going to install fresh ubuntu-rockchip to eMMC, then rsync /home /usr to nvme and mount to rootfs. still curious if anyone has any other options for preinstalled images though, to avoid rsync/fstab after initial setup.

**Comment by bagong** [comment]
Just install everything to ssd, it is much faster (at least nominally)
Best
.r.
